src/Framework/ClassResolver/AbstractClassResolver.php

```diff
@@ -24,6 +24,11 @@ abstract class AbstractClassResolver
 
     private ?InstanceCreator $instanceCreator = null;
 
+    public static function resetCache(): void
```

Comment:
What do you think about adding the flag `@internal just for testing purposes` (or something similar) to make the function not API friendly, as a warning for other people who might be interested to use it? I don't know. What do you think, @JesusValera?